hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/keygen/factory/HoodieSparkKeyGeneratorFactory.java

```diff
@@ -54,7 +59,12 @@ public static KeyGenerator createKeyGenerator(TypedProperties props) throws IOEx
   private static BuiltinKeyGenerator createKeyGeneratorByType(TypedProperties props) throws IOException {
     // Use KeyGeneratorType.SIMPLE as default keyGeneratorType
     String keyGeneratorType =
-        props.getString(HoodieWriteConfig.KEYGENERATOR_TYPE_PROP.key(), KeyGeneratorType.SIMPLE.name());
```

Comment:
Why do we need to refactor this? For adding a log message?

Reply:
> Why do we need to refactor this? For adding a log message?

Yes, to remind the user that they didn't set the key config, hudi use the default one